src/main/resources/assets/openblocks/lang/de_de.lang

```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
+openblocks.gui.credits.title=Danksagungen
+openblocks.gui.credits.content=§LBoq, Mikee und NeverCast§R - Führende Entwickler von OpenBlocks (Dürfen wir uns selbst hier nennen?)\n\n§LSinZ und Yoshi2§R - Von Anfang an Mitglieder des OpenMods-Teams.\n\n§LVexatos und crafteverywhere§R -Übersetzen regelmäßig die Modifikation.\n\nJeder, der innerhalb der letzten zwei Jahre zum Code beigetragen hat.\n\n§LDirewolf20 und der Rest von ForgeCraft§R - Für das Entdecken der Fehler und Abstürze vor der Veröffentlichung einer Version.\n\nJeder, der einen Einblick oder Let's Play machte sowie generell die Modifikation untertützt hat, seit wir begannen!
 
-openblocks.gui.config.title=Einfache Config-Bearbeitung
-openblocks.gui.config.content=OpenMods erlaubt, sowohl Client- als auch Serverkonfigurationen über den Chat zu ändern. Manche brauchen nicht mal einen Neustart!\n\nServerkonfigurationen können mit '§L/om_config_s§R' (setzt Operator-Rechte vorraus) und Clientkonfigurationen mit '§L/om_config_c§R' geändert werden.\n\nZum Beispiel:\n\n§L/om_config_s set OpenBlocks dropblock elevatorDrainsXP false§R\n§L/om_config_s save§R\n\nFür eine Liste aller verfügbaren Kommandos benutze: §L/help om_config_s§R oder §L/help om_config_c§R
+openblocks.gui.config.title=Einfache Konfigurations-Bearbeitung
+openblocks.gui.config.content=OpenMods erlaubt, sowohl den Client- als auch den Serverkonfigurationen über den Chat zu ändern. Manche brauchen nicht mal einen Neustart!\n\nServerkonfigurationen können mit '§L/om_config_s§R' (setzt Operator-Rechte vorraus) und Clientkonfigurationen mit '§L/om_config_c§R' geändert werden.\n\nZum Beispiel:\n\n§L/om_config_s set OpenBlocks dropblock elevatorDrainsXP false§R\n§L/om_config_s save§R\n\nFür eine Liste aller verfügbaren Befehlen benutze: §L/help om_config_s§R oder §L/help om_config_c§R
 
-openblocks.gui.restore_inv.title=Inventar-Backup
-openblocks.gui.restore_inv.content=Jeder kann einen Fehler machen, aber mit dem §L/ob_inventory§R-Befehl kannst du einige von ihnen wieder beheben. Dieser Befehl kann zum Wiederherstellen des Inventars genutzt werden. Ein Backup wird erstellt wenn:\n - der Spieler stirbt\n - ein Grabstein gespawnt wird \n - der Spieler den Befehl §L/ob_inventory store§R eingibt\n\nTo restore an inventory you need the name of the backup. Gehen in den unten genannten Ordner oder den saves-Ordner bei Servern und such nach Dateien mit dem Namen §Linventory-*.dat§R. Dann stelle das Inventar wieder her mit §L/ob_inventory restore <player> <id>§R - wobei id der volle Dateiname ist, oder nur der mittlere Teil ohne §Linventory-§R and §L.dat§R. Du kannst Tabvervollständigung benutzen. Backups sind normal NBT-Dateien und können mit einem entsprechenden Editor geöffnet werden.
+openblocks.gui.restore_inv.title=Inventar-Sicherung
```

Comment:
Not sure this is good to translate.

Reply:
I know "Sicherung" is sadly a german word.

Reply:
So what? There is no need to translate the word "Backup".

Reply:
But "Sicherung" is german and "Backup" is english.